data/filetypes.common

```diff
@@ -106,6 +106,7 @@ error=0xff0000;0xBFBFBF;false;italic
 
 selection=0x000000;0xc0c0c0;false;true
 current_line=0x000000;0xf0f0f0;true;
+current_word=0xffffff
```

Comment:
It's kind of "ah crap I can't get this working, so I'll leave it alone". I tried a bunch of fg vs bg and changing the colors and nothing seemed to be working correctly. I think, if it is somehow working it's because the code uses `invert()` maybe.